Sources/PostgresKit/PostgresRow+SQL.swift

```diff
@@ -1,8 +1,17 @@
-extension PostgresRow: SQLRow {
-    public func decode<D>(column: String, as type: D.Type) throws -> D where D : Decodable {
-        guard let data = self.column(column) else {
+extension PostgresRow {
+    public func sql(using decoder: PostgresDataDecoder) -> SQLRow {
```

Comment:
```suggestion
    public func sql(decoder: PostgresDataDecoder = .init()) -> SQLRow {
```

Reply:
We should have a default here. We should also use `decoder` as the label for consistency since the similar method on database is `sql(encoder:decoder:)`